### DISCUSSION TITLE
SPEF extraction in interactive mode

### DISCUSSION THREAD

**Discussion by akhilesh911** [The-OpenROAD-Project/OpenLane#1923] [Q&A]
Good morning,
I am trying to run the OpenLane flow in interactive mode. I have post-routing .odb file of my design. Now I want to execute further steps. I have written netlist, powered netlist, def. With command, the error is as shown in the log file.
Kindly suggest the solutions if any.
[signoff.zip](https://github.com/The-OpenROAD-Project/OpenLane/files/12224783/signoff.zip)

**Comment by vijayank88** [comment]
@akhilesh911 
Upload a test case to reproduce the error and steps you tried in interactive.

**Reply by akhilesh911** [reply]
[issue_reproducible_1.zip](https://github.com/The-OpenROAD-Project/OpenLane/files/12225180/issue_reproducible_1.zip)

**Reply by kareefardi** [reply]
Something went wrong in the flow, where the merged lef lost the information of the metal layers. Can you share your interactive script (or the commands used interactively) ?

**Reply by akhilesh911** [reply]
I have used commands after obtaining routing odb.
1) prep -design picosoc -tag run16
2) set_netlist designs/picosoc/runs/run16/results/routing/picosoc.pnl.v
3) set_def designs/picosoc/runs/run16/results/routing/picosoc.def
4) run_parasitics_sta

I want to fix max cap, slew. i have replaced some cells and now performing sta. Am I doing something wrong?
Please let me know the steps if I am wrong.
